public/js/maineditor.js

```diff
@@ -0,0 +1,206 @@
+const select = function(selector) {
```

Comment:
The file should be named `mainEditor.js`

Reply:
Done!